File: app/src/main/java/com/adrianiglesia/atqr/View/MainActivity.java
package com.adrianiglesia.atqr.View;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.adrianiglesia.atqr.R;

import butterknife.ButterKnife;


public class MainActivity extends AppCompatActivity {

    private static int TIME = 2000;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(MainActivity.this, LoginActivity.class);
                startActivity(intent);
                finish();

            }
        },TIME);

    }

}

File: app/src/main/java/com/adrianiglesia/atqr/view/MainActivity.kt
package com.adrianiglesia.atqr.view

import android.content.Intent
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.adrianiglesia.atqr.R

import butterknife.ButterKnife


class MainActivity : AppCompatActivity() {

    private val TIME = 500
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        ButterKnife.bind(this)


        Handler().postDelayed({
            val intent = Intent(this@MainActivity, LoginActivity::class.java)
            startActivity(intent)
            finish()
        }, TIME.toLong())

    }



}
